src/css/components/_progress.scss

```diff
@@ -15,6 +15,7 @@
 .video-js .vjs-progress-control {
   @include flex(auto);
   @include display-flex(center);
+  margin: 0 0.45em 0 0.45em;
```

Comment:
Actually, you're right. There is a slight difference.
The issue is that with this line here, our tooltips look like this:
![broken tooltip](https://cldup.com/qcRJ6O_tEW.png)
notice how the right side is a bit cut off. Vs, this:
![working tooltip](https://cldup.com/qvmgvAinrD.png)

The position of the mouse marker in both those cases is correct but the tooltip isn't.